src/pages/billing/Project.js

```diff
@@ -393,41 +401,21 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
                 }
               })
             ])])
-          ])
-        ]
-      ),
-      div(
-        {
-          style: { display: 'grid', rowGap: '1.25rem' },
-          'aria-atomic': true,
-          'aria-live': 'polite',
-          'aria-busy': !isProjectCostReady
-        }, [
-          div(
-            {
-              style: {
-                display: 'grid',
-                gridTemplateColumns: 'repeat(3, minmax(max-content, 1fr))',
-                columnGap: '1.25rem'
-              }
-            },
-            [
-              ...(_.map(name => h(CostCard, {
-                title: `Total ${name}`,
-                amount: (!isProjectCostReady ? '...' : projectCost[name]),
-                showAsterisk: name === 'spend',
-                key: name
-              }),
-              ['spend', 'compute', 'storage'])
-              )
-            ]
-          ),
-          h(OtherMessaging, { cost: isProjectCostReady ? projectCost['other'] : null })
+          ]),
+          ...(_.map(name => CostCard({
```

Comment:
@msilva-broad I thought there were good reasons to keep CostCard as a component. Does it cause issues as a component?

Reply:
I thought so too, but as I reverted to the original setup, I was still seeing some wierdness and was tired of fussing with it all... reverting fixed things.  so, meh.